identity/backend/src/main/resources/application.yaml

```diff
@@ -13,6 +13,10 @@ spring:
       accept-case-insensitive-enums: true
     deserialization:
       read-unknown-enum-values-using-default-value: true
+  jpa:
+   generate-ddl: false
+   hibernate:
+    ddl-auto: none
```

Comment:
May I suggest to use `validate` for `ddl-auto`? It could help us find potential mapping issues between entities and tables as early as possible 🙂 wdyt?